docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.2"
```

Comment:
Is this file an example? If that's the case I would move the contents to the README, indicating that this is an expected output (given certain input). Alternatively, move the launch.json and this file to a folder called `samples`.